src/main/resources/assets/js/application.js

```diff
@@ -0,0 +1,25 @@
+function getHostName(url) {
+  var match = url.match(/:\/\/(www[0-9]?\.)?(.[^/:]+)/i);
```

Comment:
I think this whole part is kind of overcomplicated for what it needs to be.

The rules are of you have a gov.uk domain then don't use the crests and font. You're on a page so what you get from url will be a url, you don't need to verify that I don't think, and really all you wanna do if find out if gov.uk is in the url and then add a css class. I just think this could be simplified a lot personally. The test case where you have gov.uk.openregister.org in my opinion is pretty unlikely and within our control.